mmcv/runner/hooks/checkpoint.py

```diff
@@ -26,6 +26,7 @@ class CheckpointHook(Hook):
             Default: -1, which means unlimited.
         sync_buffer (bool): Whether to synchronize buffers in different
             gpus. Default: False.
+        keep_last (bool): Whether to save last checkpoint.
```

Comment:
save_last (bool): Whether to force the last checkpoint to be saved regardless of interval.